### PR TITLE
Fix TRAPI 1.5 critical validation errors regarding missing attributes

### PR DESCRIPTION
**Pull request by CaseyTa**
Fix TRAPI 1.5 critical validation errors regarding missing attributes in (KG) nodes and results node bindings and edge bindings

In TRAPI 1.5, attributes are now required and expected to be non-null but can have 0 items.

### CODE REVIEW

**Comment by CaseyTa** [comment]
@micheldumontier @renayang2023 Could you please review? OpenPredict is failing integration with CQS after the call from Workflow Runner due to "critical" TRAPI validation errors in OpenPredict's response. Changes have been tested on my local deployment of OpenPredict.

**Review by renayang2023 — APPROVED** [review]

**Comment by renayang2023** [comment]
> @micheldumontier @renayang2023 Could you please review? OpenPredict is failing integration with CQS after the call from Workflow Runner due to "critical" TRAPI validation errors in OpenPredict's response. Changes have been tested on my local deployment of OpenPredict.

@CaseyTa Thanks, Casey. I approved the changes. It seems that one workflow is still awaiting approval from those with write access to merge pull requests.

**Pull request merged**
